.github/workflows/arm64.yml

```diff
@@ -83,6 +83,19 @@ jobs:
             --ignore=tests/scenarios/test_sync.py \
             --ignore=tests/scenarios/test_transformation.py \
             -k "not test_search"
+      - run: echo "TEST_FAILURE=true" >> $GITHUB_ENV
+        if: failure()
+      - uses: actions/checkout@v3
+        with:
+          persist-credentials: false
+      - name: Create JIRA issue
```

Comment:
Let's add double quotes around the name for consistency with the rest of the workflow.